AutomatedTesting/Gem/PythonTests/scripting/Node_HappyPath_DuplicateNode.py

```diff
@@ -34,80 +46,67 @@ def Node_HappyPath_DuplicateNode():
 
     :return: None
     """
-    from PySide2 import QtWidgets, QtTest
-    from PySide2.QtCore import Qt
-
-    from editor_python_test_tools.utils import Report
-    from editor_python_test_tools.utils import TestHelper as helper
-    import editor_python_test_tools.pyside_utils as pyside_utils
-
-    import azlmbr.legacy.general as general
-
-    WAIT_FRAMES = 200
 
-    NODE_NAME = "Print"
-    NODE_CATEGORY = "Debug"
-    EXPECTED_STRING = f"{NODE_NAME} - {NODE_CATEGORY} (2 Selected)"
 
-    def command_line_input(command_str):
-        cmd_action = pyside_utils.find_child_by_pattern(
-            sc_main, {"objectName": "action_ViewCommandLine", "type": QtWidgets.QAction}
+    def run_command_line_input(self, sc_editor, sc_editor_main_window, command_str):
+        command_line_action = pyside_utils.find_child_by_pattern(
+            sc_editor_main_window, {"objectName": "action_ViewCommandLine", "type": QtWidgets.QAction}
         )
-        cmd_action.trigger()
-        textbox = sc.findChild(QtWidgets.QLineEdit, "commandText")
+        command_line_action.trigger()
+        textbox = sc_editor.findChild(QtWidgets.QLineEdit, "commandText")
         QtTest.QTest.keyClicks(textbox, command_str)
         QtTest.QTest.keyClick(textbox, Qt.Key_Enter, Qt.NoModifier)
 
-    def grab_title_text():
-        scroll_area = node_inspector.findChild(QtWidgets.QScrollArea, "")
-        QtTest.QTest.keyClick(graph, "a", Qt.ControlModifier, WAIT_FRAMES)
+    def grab_title_text(self, sc_graph_node_inspector, sc_graph):
+        scroll_area = sc_graph_node_inspector.findChild(QtWidgets.QScrollArea, "")
+        QtTest.QTest.keyClick(sc_graph, "a", Qt.ControlModifier, WAIT_FRAMES)
         background = scroll_area.findChild(QtWidgets.QFrame, "Background")
-        title = background.findChild(QtWidgets.QLabel, "Title")
-        text = title.findChild(QtWidgets.QLabel, "Title")
+        title = background.findChild(QtWidgets.QLabel, TITLE_STRING)
+        text = title.findChild(QtWidgets.QLabel, TITLE_STRING)
         return text.text()
```

Comment:
This could use a more meaningful name such as `get_node_title_text` and possibly moved to the sc utils class to help define the context of the function even further.

Reply:
moved this behavior to the sc utils class. updated the names of the variables being used as well so it's clearer what's happening